MediaBrowser.WebDashboard/Api/DashboardService.cs

```diff
@@ -149,7 +149,7 @@ public string DashboardUIPath
                     return _serverConfigurationManager.Configuration.DashboardSourcePath;
                 }
 
-                return Path.Combine(_serverConfigurationManager.ApplicationPaths.ApplicationResourcesPath, "jellyfin-web", "src");
+                return Path.Combine(_serverConfigurationManager.ApplicationPaths.WebPath, "src");
```

Comment:
again, `src` should be part of `.WebPath`

Reply:
Done.